ui/app/components/auth/form-template.hbs

```diff
@@ -36,18 +36,32 @@
           {{/if}}
         </:back>
 
-        {{! TABS OR DROPDOWN }}
+        {{! DIRECT LINK, TABS OR DROPDOWN }}
         <:authSelectOptions>
           <div class="has-bottom-margin-m">
-            {{#if this.renderTabs}}
-              <Auth::Tabs
-                @authTabData={{@authTabData}}
-                @authTabTypes={{this.authTabTypes}}
-                @displayNameHelper={{this.displayName}}
-                @handleTabClick={{this.setAuthType}}
-                @selectedAuthMethod={{this.selectedAuthMethod}}
-              />
+            {{#if this.showCustomAuthOptions}}
```

Comment:
does this naming make sense to folks? really struggled with how to make this intuitive...

Reply:
Works for me!